### PR TITLE
Feature - Add Weapons ID

### PR DESCRIPTION
**Pull request by felpsds**
I've addes the mentioned on #135.

### CODE REVIEW

**Comment by ByMykel** [comment]
Hi, thanks for the PR, but I am using `getWeaponName` in some places and I asume that the value returned is a string.

If the idea is to return that number as the id we can just replace this line:

```diff
 weapon: { 
-     id: weapon, 
+     id: weaponIDMapping[weapon], 
     name: translatedName, 
 }, 
```

In these two files:

https://github.com/ByMykel/CSGO-API/blob/bd73dd0e568defd51f7dff2e503281c19b97feff/services/skins.js#L113-L116
https://github.com/ByMykel/CSGO-API/blob/bd73dd0e568defd51f7dff2e503281c19b97feff/services/skinsNotGrouped.js#L155-L158

Let me know what you think.

**Comment by felpsds** [comment]
This is good for me, but considering that someone might be using id: "weapon_deagle" as part of a primary key (PK) or mapping guns, this change could break things.

I suggest doing something like this instead:
```diff
weapon: {
      id: weapon,
+     weapon_id: weaponIDMapping[weapon],
      name: translatedName,
},

Let me know if this works for you, and I’ll create a new commit.

**Comment by ByMykel** [comment]
> This is good for me, but considering that someone might be using id: `"weapon_deagle"` as part of a primary key (PK) or mapping guns, this change could break things.
> 
> I suggest doing something like this instead:
> 
> ```diff
> weapon: {
>       id: weapon,
> +     weapon_id: weaponIDMapping[weapon],
>       name: translatedName,
> },
> 
> Let me know if this works for you, and I’ll create a new commit.
> ```

Yeah, this would be great. Thanks!

**Comment by felpsds** [comment]
Changes done on last commit.
@ByMykel

**Comment by ByMykel** [comment]
@felpsds looks great.

While I was testing it I have forgot to mentione these two places for knives:

https://github.com/ByMykel/CSGO-API/blob/bd73dd0e568defd51f7dff2e503281c19b97feff/services/skins.js#L180-L183
https://github.com/ByMykel/CSGO-API/blob/bd73dd0e568defd51f7dff2e503281c19b97feff/services/skinsNotGrouped.js#L245-L248

**Comment by felpsds** [comment]
@ByMykel Done!

File: services/skins.js [hunk 1]
```diff
@@ -179,6 +181,7 @@ export const getSkins = () => {
             description: $t(knife.item_description),
             weapon: {
                 id: knife.item_name,
+                weapon_id: weaponIDMapping[weapon],
```
Comment (ByMykel) [inline services/skins.js:184]:
```suggestion
                weapon_id: weaponIDMapping[knife.name],
```

File: services/skinsNotGrouped.js [hunk 2]
```diff
@@ -244,6 +246,7 @@ export const getSkinsNotGrouped = () => {
                     }),
                     weapon: {
                         id: knife.item_name,
+                        weapon_id: weaponIDMapping[weapon],
```
Comment (ByMykel) [inline services/skinsNotGrouped.js:249]:
```suggestion
                        weapon_id: weaponIDMapping[knife.name],
```

**Review by ByMykel — COMMENTED** [review]
These two changes and we can merge this.

**Review by ByMykel — APPROVED** [review]
LGTM

**Pull request merged**